CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
```

Comment:
`[bug]` is so negative what about `[fix]`? Also I think `[change]` doesn't add any value. Of course it was a change, that's why its called *Changelog*. **nag, nag, nag** :wink: 

My proposal:
 * `[feature]` -> new feature
 * `[fix]` -> bug fix
 * `[info]` -> random note
 * `[breaking]` -> breaking change